app/routes/cancellation/api.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * Copyright 2023-2024 Google LLC
```

Comment:
Can you please update the copyright on the new files to be 2025? We don't need to do this for old files, so just the new ones in this PR.